### ISSUE TITLE
Possible user/configuration issue

### ISSUE THREAD

**Issue by jordanleven** [jeremyvii/vs-docblockr#40]
Hey guys,

I'm trying to migrate over to VS Code from Sublime and was looking for a DocBlock package and found yours. But when I try to add blocks to a class method, like this:
```
public function myTestMethod( string $myString, bool $myBool ){

 }
```
by typing above the method name, it only adds the empty comments, like this:
```
  /**
   * 
   */
  public function myTestMethod( string $myString, bool $myBool ){

  }
```

If, however, I start typing on the same line as the method, I get this:
```
 /**
   * [ description]
   *
   * @return  [type]  [return description]
   */public function myTestMethod( string $myString, bool $myBool ){

  }
```

But it doesn't include any params or anything. Any ideas what I'm doing wrong? Thanks in advance!

**Comment by jeremyvii** [comment]
The example you provide should be working without issue. I just tested following which does work:

``` php
public function myTestMethod(string $myString, bool $myBool) {

}
```

This should an easy fix, but as a work around you can remove the first and last spaces in the arguments list.

**Comment by jordanleven** [comment]
Thanks for the quick reply! I did try removing the spaces, but I still am getting the same error. I'm attaching a screen capture of what I'm talking about.
![Untitled](https://user-images.githubusercontent.com/7015466/59782764-b0280400-928c-11e9-9cab-00127517b65a.gif)

**Comment by jordanleven** [comment]
Hey @jeremyvii - just an FYI I get the same issue on JS and Java as well (that's why I thought it was some type of configuration issue or something).

**Comment by jeremyvii** [comment]
Aw I see. I was incorrect. It's not an issue with spaces around the parameters, but an issue with the lack of space between the ending parenthesis and the opening curly bracket `){`. Given that it appears to be in an error in multiple languages, the bug is likely in [the lexer](https://github.com/jeremyvii/vs-docblockr/blob/master/src/lexer.ts). 

I can say for sure it is nothing configuration related on your end.

**Comment by jordanleven** [comment]
Ah, you're right. After adding the space between the closing parenthesis and the opening curly bracket, it seems to be working.

**Comment by cn-zx** [comment]
I also encountered this problem, in sublime,
function test(argument) {
      // body...
    }
This simple function, after typing /**, can trigger comments using both tab and enter, and is normal.
But in vs, after the enter, only /**
      *

**Comment by jeremyvii** [comment]
@cn-zx please make a separate issue for what you've encountered, as it is unrelated to this issue.

**Comment by jeremyvii** [comment]
This should be fixed with the release of [0.5.8](https://github.com/jeremyvii/vs-docblockr/releases/tag/0.5.8).
